### PR TITLE
Update Essentials API Docs Misc

### PR DESCRIPTION
**Pull request by jfversluis**
### Description of Change

Moves the external comments to inline and minor improvements. This should be (one of) the last for Essentials before we can enable the build errors for missing comments.

### Issues Fixed

Related to #3960

### CODE REVIEW

File: src/Essentials/src/FileSystem/FileSystem.shared.cs [hunk 1]
```diff
@@ -67,7 +67,7 @@ public static Task<Stream> OpenAppPackageFileAsync(string filename)
 		/// <summary>
 		/// Determines whether or not a file exists in the app package.
 		/// </summary>
-		/// <param name="filename">The name of the file (excluding the path) to load from the app package.</param>
+		/// <param name="filename">The name of the file (excluding the path) to check for existence</param>
```
Comment (mattleibow) [inline src/Essentials/src/FileSystem/FileSystem.shared.cs:70]:
```suggestion
		/// <param name="filename">The path of the file (relative to the app package) to check the existence of.</param>
```

File: src/Essentials/src/PhoneDialer/PhoneDialer.shared.cs [hunk 2]
```diff
@@ -50,6 +50,10 @@ public static class PhoneDialer
 		public static void Open(string number)
 			=> Current.Open(number);
 
+		/// <summary>
+		/// Provides the default implementation for static usage of this API.
+		/// This is the same as <see cref="Default"/>.
```
Comment (mattleibow) [inline src/Essentials/src/PhoneDialer/PhoneDialer.shared.cs:55]: Maybe not this one as Default needs to be removed as it was a bad merge. We can probably obsolete/hide it now already.

Comment (jfversluis) [inline src/Essentials/src/PhoneDialer/PhoneDialer.shared.cs:55]:
```suggestion
```

File: src/Essentials/src/UnitConverters/UnitConverters.shared.cs [hunk 3]
```diff
@@ -23,111 +25,225 @@ public static class UnitConverters
...
 			pascals / atmospherePascals;
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/UnitConverters.xml" path="//Member[@MemberName='CoordinatesToMiles']/Docs/*" />
+		/// <summary>
+		/// Calculates the distance between two coordinates�in miles.
+		/// </summary>
+		/// <param name="lat1">First latitude.</param>
+        /// <param name="lon1">First longitude.</param>
+        /// <param name="lat2">Second latitude.</param>
+        /// <param name="lon2">Second longitude.</param>
```
Comment (mattleibow) [inline src/Essentials/src/UnitConverters/UnitConverters.shared.cs:234]:
```suggestion
		/// <param name="lon1">First longitude.</param>
		/// <param name="lat2">Second latitude.</param>
		/// <param name="lon2">Second longitude.</param>
```

File: src/Essentials/src/WebAuthenticator/WebAuthenticator.shared.cs [hunk 4]
```diff
@@ -6,16 +6,37 @@
...
+		/// Opens the specified URI to start the authentication flow.
+		/// </summary>
+		/// <param name="uri">The URI to open that will start the authentication flow.</param>
+		/// <returns><see langword="true"/> when the URI has been opened, otherwise <see langword="false"/>.</returns>
 		bool OpenUrlCallback(Uri uri);
 #elif ANDROID
+		/// <summary>
+		/// The event that is triggered when an authentication flow calls back into the Android application.
+		/// </summary>
+		/// <param name="intent">A <see cref="Android.Content.Intent"/> object containing additional data about this resume operation.</param>
```
Comment (mattleibow) [inline src/Essentials/src/WebAuthenticator/WebAuthenticator.shared.cs:38]:
```suggestion
		/// <param name="intent">An <see cref="Android.Content.Intent"/> object containing additional data about this resume operation.</param>
```

File: src/Essentials/src/Types/DisplayInfo.shared.cs [hunk 5]
```diff
@@ -28,47 +45,88 @@ public DisplayInfo(double width, double height, double density, DisplayOrientati
...
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/DisplayInfo.xml" path="//Member[@MemberName='Density']/Docs/*" />
+		/// <summary>
+		/// Gets a value representing the screen density.
+		/// </summary>
+		/// <remarks>
+        /// <para>The density is the scaling or a factor that can be used to convert between physical pixels and scaled pixels. For example, on high resolution displays, the physical number of pixels increases, but the scaled pixels remain the same.</para>
+        /// <para>In a practical example for iOS, the Retina display will have a density of 2.0 or 3.0, but the units used to lay out a view does not change much. A view with a UI width of 100 may be 100 physical pixels (density = 1) on a non-Retina device, but be 200 physical pixels (density = 2) on a Retina device.</para>
+        /// <para>On Windows, the density works similarly, and may often relate to the scale used in the display. On some monitors, the scale is set to 100% (density = 1), but on other high resolution monitors, the scale may be set to 200% (density = 2) or even 250% (density = 2.5).</para>
+        /// </remarks>
```
Comment (mattleibow) [inline src/Essentials/src/Types/DisplayInfo.shared.cs:65]:
```suggestion
		/// <para>The density is the scaling or a factor that can be used to convert between physical pixels and scaled pixels. For example, on high resolution displays, the physical number of pixels increases, but the scaled pixels remain the same.</para>
		/// <para>In a practical example for iOS, the Retina display will have a density of 2.0 or 3.0, but the units used to lay out a view does not change much. A view with a UI width of 100 may be 100 physical pixels (density = 1) on a non-Retina device, but be 200 physical pixels (density = 2) on a Retina device.</para>
		/// <para>On Windows, the density works similarly, and may often relate to the scale used in the display. On some monitors, the scale is set to 100% (density = 1), but on other high resolution monitors, the scale may be set to 200% (density = 2) or even 250% (density = 2.5).</para>
		/// </remarks>
```

File: src/Essentials/src/Types/DisplayInfo.shared.cs [hunk 6]
```diff
@@ -28,47 +45,88 @@ public DisplayInfo(double width, double height, double density, DisplayOrientati
...
 		public DisplayRotation Rotation { get; }
 
-		/// <include file="../../docs/Microsoft.Maui.Essentials/DisplayInfo.xml" path="//Member[@MemberName='RefreshRate']/Docs/*" />
+		/// <summary>
+		/// Gets the refresh rate (in Hertz) of the device's display.
+		/// </summary>
 		public float RefreshRate { get; }
 
+		/// <summary>
+		///	Equality operator for equals.
```
Comment (mattleibow) [inline src/Essentials/src/Types/DisplayInfo.shared.cs:84]:
```suggestion
		/// Equality operator for equals.
```

**Review by StephaneDelcroix — APPROVED** [review]

**Pull request merged**